docs/docs-content/clusters/edge/local-ui/cluster-management/export-cluster-definition.md

```diff
@@ -10,11 +10,33 @@ tags: ["edge"]
 You can export cluster definitions from a cluster in Palette and use the definition to provision a cluster in an Edge
 host. A cluster definition contains one or more cluster profiles, including the profile variables used in the profiles.
 
+You can use either the Palette Edge CLI to export the cluster definition or use the Palette API. The CLI offers a more
+user-friendly interface, but requires a machine with X86_64 architecture. If you are on an ARM64 machine such as Apple
+Silicon, use the API approach instead.
+
 :::preview
 
 :::
 
-## Prerequisites
+## Export Cluster Definition with Palette Edge CLI
+
+### Prerequisites
+
+- At least one [cluster profile](/docs/docs-content/profiles/cluster-profiles/cluster-profiles.md) in Palette.
+
+- A Palette API key. For more information on how to create a Palette API key, refer to
+  [Create API Key](/docs/docs-content/user-management/authentication/api-key/create-api-key.md).
+
+- A physical or virtual Linux machine with _AMD64_ (also known as _x86_64_) processor architecture. You can issue the
+  following command in the terminal to check your processor architecture.
+
+  ```bash
+  uname -m
+  ```
+
+## Export Cluster Definition with Palette API
```

Comment:
Something odd is happening here. Did you mean to use tabs?

Reply:
Ah, my bad 🤦 i think there was one section that was out of order.